docs/myriad-configuration.md

```diff
@@ -26,6 +26,12 @@ frameworkName: MyriadAlpha
 # Myriad's mesos framework role.
 frameworkRole: someRoleName
 
+# Myriad's mesos framework user (Defaults to user running the resource manager if absent, necessary for remote distribution).
+frameworkUser: someUserName
+
+# Myriad's mesos framework super user (Necessary only for remote distribution, otherwise ignored).
```

Comment:
You could be a little more specific and explain that this is the user that extracts and chowns the (hadoop?) binaries over to the frameworkUser.